static/app/views/onboarding/targetedOnboarding/components/createProjectsFooter.tsx

```diff
@@ -42,31 +42,29 @@ export default function CreateProjectsFooter({
 }: Props) {
   const api = useApi();
   const {teams} = useTeams();
+  const [clientState, setClientState] = useOnboardingState();
 
   const createProjects = async () => {
+    if (!clientState) {
```

Comment:
@Neo-Zhixing we should disable the button until the client state is ready if this is a problem

Reply:
In these cases, it's worth considering changing usePersistedStore to instead of returning state, to also return something like {isLoading, isReady, store} = usePersistedStore. The null state here implicitly means loading, which can be a "gotcha" in the future, but that could be an improvement for another time :)